skyportal/utils/cache.py

```diff
@@ -111,6 +111,26 @@ def __setitem__(self, name, data):
 
         self.clean_cache()
 
+    def __delitem__(self, name):
+        """Remove item from the cache.
+
+        Parameters
+        ----------
+        name : str
+            Name of item to remove.
+        """
+        if name is None:
+            return
+
+        fn = self._hash_filename(name)
+        if not fn.exists():
+            return
+        os.remove(fn)
```

Comment:
I would have a try except here maybe, just to avoid an edge case where it gets deleted exactly between when you run your if statement above and when you delete it, or if for some reason we don't have delete rights.

Reply:
✅ Done.